### DISCUSSION TITLE
`brew install --cask cocoapods` giving error

### DISCUSSION THREAD

**Discussion by kushal98** [Homebrew/discussions#1504] [Getting started]
==> Caveats
cocoapods has been officially discontinued upstream.
It may stop working correctly (or at all) in recent versions of macOS.

==> Downloading https://github.com/CocoaPods/CocoaPods-app/releases/download/1.5.2/CocoaPods.app-1.5
Warning: Transient problem: timeout Will retry in 1 seconds. 3 retries left.
Warning: Transient problem: timeout Will retry in 2 seconds. 2 retries left.  
Warning: Transient problem: timeout Will retry in 4 seconds. 1 retries left.  
##O=#  #                                                                      
curl: (6) Could not resolve host: github.com
Error: Download failed on Cask 'cocoapods' with message: Download failed: https://github.com/CocoaPods/CocoaPods-app/releases/download/1.5.2/CocoaPods.app-1.5.2.tar.bz2

**Comment by SMillerDev** [comment]
```
 curl: (6) Could not resolve host: github.com
```
It seems you have a general issue connecting to github. Please resolve this and try again.
